src/sonic-py-common/tests/interface_test.py

```diff
@@ -0,0 +1,56 @@
+import os
+import sys
+
+# TODO: Remove this if/else block once we no longer support Python 2
+if sys.version_info.major == 3:
+    from unittest import mock
+else:
+    # Expect the 'mock' package for python 2
+    # https://pypi.python.org/pypi/mock
+    import mock
```

Comment:
It appears this is not being used in this file. Suggest to remove. We can add it back if we need to do mocking in this file at a later time.